test/box-luatest/gh_8604_support_square_brackets_in_procedure_resolution_for_lua_calls_test.lua

```diff
@@ -0,0 +1,117 @@
+local server = require('luatest.server')
+local t = require('luatest')
+
+local g = t.group()
+g.before_all(function(cg)
+    cg.server = server:new()
+    cg.server:start()
+    cg.server:exec(function()
```

Comment:
Nit: AFAIC, you don't need to create a server for this test.

Reply:
Thanks, rewrote the test to use main instance.